searchapp/utils/utils.py

```diff
@@ -75,20 +77,23 @@ def convert_marker_data(marker_path, subject_breakup):
             question_type = header_row[question_type_start_row].value
             while (question_type != None):
                 #  Incase you get a string like '1A' as your question type, thats fine, else if you get a float like 2.0, convert to '2'
-                question_type = question_type if type(question_type) == str else str(int(question_type))
+                question_type = question_type if type(
+                    question_type) == str else str(int(question_type))
                 total_questions, questions_to_attempt = subject_breakup[question_type]
                 questions_of_given_type = []
                 for i in range(question_type_start_row, question_type_start_row + total_questions):
                     question_number = i - question_type_start_row + 1
                     chapter_number = int(chapter_row[i].value)
                     marks_secured = row[i].value or 0.0
-                    questions_of_given_type.append((marks_secured, question_number, chapter_number))
+                    questions_of_given_type.append(
+                        (marks_secured, question_number, chapter_number))
                 questions_of_given_type.sort(key=lambda x: x[0], reverse=True)
                 questions_of_given_type = questions_of_given_type[:questions_to_attempt]
                 student_to_answer[student][question_type] = questions_of_given_type
                 question_type_start_row += total_questions
                 question_type = header_row[question_type_start_row].value
-    return student_to_answer, list(set([int(val.value) for val in chapter_row if type(val.value) == type(1.0)]))
+    return student_to_answer, list(set([int(val.value) for val in chapter_row if type(val.value) == type(1)]))
```

Comment:
`type(1)` and `type(1.0)` are not the same. Was this change intended?

Reply:
It was flaky as it did not work for me but worked for Sandhya. I have reverted this change now.